### PR TITLE
192 scalar scaler

### LINKED ISSUE #192: Scalar scaler
Need a scalar version of `VectorScaler` because the use case is common and passing a number to `VectorScaler` results in an error.  Also the name `ScalarScaler` brings me joy.

something like 
```python
class ScalarScaler:
    """
    Applies a scalar scale factor

    This class conforms to `Scaler` protocol.
    """

    def __init__(self, factor: float = 1.0) -> None:
        self._factor = factor

    def pre_scale(self, data_tensor):
        return data_tensor / self._factor

    def post_scale(self, data_tensor):
        return data_tensor * self._factor
```

### PR DESCRIPTION
**Pull request by jtencer**

### CODE REVIEW

**Comment by fnrizzi** [comment]
nice title :)

File: romtools/vector_space/utils/scaler.py [hunk 1]
```diff
@@ -169,12 +171,29 @@ def post_scale(self, data_tensor: np.ndarray) -> np.ndarray:
...
+    """
+
+    def __init__(self, factor: float = 1.0) -> None:
+        self._factor = factor
+
+    def pre_scale(self, data_tensor: np.ndarray) -> np.ndarray:
+        return data_tensor / self._factor
+
+    def post_scale(self, data_tensor: np.ndarray) -> np.ndarray:
+        return data_tensor * self._factor
```
Comment (fnrizzi) [inline romtools/vector_space/utils/scaler.py:192]: i thought we were doing operations in place, is that not true?

Comment (jtencer) [reply romtools/vector_space/utils/scaler.py:192]: I think we did decide that, but all of the other scalers look like they return a copy so maybe that didn't make its way down to the scalers?

Comment (eparish1) [reply romtools/vector_space/utils/scaler.py:192]: Yes, it looks like there are some inconsistencies, I'll make an issue.

**Review by eparish1 — APPROVED** [review]

**Pull request merged**